src/server/stream_family.cc

```diff
@@ -876,6 +911,205 @@ OpResult<vector<GroupInfo>> OpListGroups(const DbContext& db_cntx, string_view k
   return result;
 }
 
+vector<Record> GetStreamRecords(stream* s, streamID start, streamID end, int reverse,
```

Comment:
lets change `int reverse` to `bool reverse` and when calling this function, please pass false/true instead fo `0/1`

Reply:
done.